common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -578,6 +579,24 @@ def test_closed(self):
                                     due=self.yesterday_str)
         self.assertTrue(module.closed())
 
+
+    def test_public_view(self):
+        """
+        Test that public view for capa problems returns
```

Comment:
```suggestion
        Tests that public view for capa problems returns
```

Reply:
This is intentionally an imperative sentence.